### PR TITLE
[Backup] az backup recoveryconfig show: Add more parameters to support restoring as files  for SQL/SAP Hana

### PR DESCRIPTION
**Pull request by sambitratha**
**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S165.

File: src/azure-cli/azure/cli/command_modules/backup/_params.py [hunk 1]
```diff
@@ -48,14 +49,17 @@
...
 resource_id_type = CLIArgumentType(help=resource_id_help, options_list=['--resource-id'])
 policy_type = CLIArgumentType(help=policy_help, options_list=['--policy'], completer=FilesCompleter(), type=file_type)
 protectable_item_type = CLIArgumentType(help=workload_type_help, options_list=['--protectable-item-type'], arg_type=get_enum_type(allowed_protectable_item_type))
 target_server_type = CLIArgumentType(help=target_server_type_help, options_list=['--target-server-type'], arg_type=get_enum_type(allowed_protectable_item_type))
 protectable_item_name_type = CLIArgumentType(help=protectable_item_name_type_help, options_list=['--protectable-item-name'])
 diskslist_type = CLIArgumentType(nargs='+', help=diskslist_help)
+target_container_name_type = CLIArgumentType(options_list=['--target-container-name'], help=target_container_name_help)
+filepath_type = CLIArgumentType(options_list=['--filepath'], help="The path to which the DB should be restored as files.")
+from_full_rp_type = CLIArgumentType(options_list=['--from-full-rp-name'], help="Name of the starting Recovery point.")
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/backup/_params.py]: can we consider another parameter name for "from_full_rp_name"? --from-full-rp-name is not very user-friendly.

**Comment by Juliehzl** [comment]
please correct me if the PR title is not very suitable.

File: src/azure-cli/azure/cli/command_modules/backup/_params.py (hunk 1 above)
Comment (sambitratha) [reply src/azure-cli/azure/cli/command_modules/backup/_params.py]: This was accepted in the Cmdlet review, Adding @pvrk for reference. The parameter name is similar to that of existing Powershell Cmdlet.

**Comment by sambitratha** [comment]
@Juliehzl The title looks fine to me

**Comment by Juliehzl** [comment]
Need to remove additional requirement.txt, but I don't have the permission to do that for your PR. Will merge first and remove these files.

**Pull request merged**